sycl/include/CL/__spirv/spirv_ops.hpp

```diff
@@ -563,10 +563,6 @@ template <typename dataT>
 extern SYCL_EXTERNAL WPipeTy<dataT> __spirv_CreatePipeFromPipeStorage_write(
     const ConstantPipeStorage *Storage) noexcept;
 
-extern SYCL_EXTERNAL void
-__spirv_ocl_prefetch(const __attribute__((opencl_global)) char *Ptr,
```

Comment:
Can you please explain why this func is not needed anymore?
It also breaks ABI and we still can't break it.

Reply:
The function is now correctly generated by clang/lib/Sema/SPIRVBuiltins.td with the right types, whereas it was generated with `signed char` before. Therefore, this declaration is no longer needed.

Reply:
OK, thanks. We should keep it as w/o it ABI should break (as the declaration has `SYCL_EXTERNAL`). I'm running checks below which should show the ABI breakage.

Reply:
I vaguely remember it complaining about some conflicts when both were present, but I am happy to revisit if the tests show ABI breakage.

Reply:
Regarding ABI I somehow mixed up `SYCL_EXTERNAL` and `SYCL_EXPORT`. The change should be ok from ABI point-of-view.